### DISCUSSION TITLE
MongoDB server is not working for me [HW2]

### DISCUSSION THREAD

**Discussion by mrpositron** [prosyslab-classroom/cs348-information-security#734] [Q&A]
I am monitoring traffic on **Attacker** and I see that I am _only_ sending packets. I monitor packets on **Amp** and I see that the size of packets sent is bigger than received. I am also monitoring packets arriving on the **Victim** side, and I see that it _only_ receives packets. However, `db.attack_session.find({})` is returning nothing. I am monitoring traffic through the network information on each virtual machine.

**P.S. Is it possible to get rid of @gyusss messages?**

**Comment by mrpositron** [comment]
Could you look at it @Hwi-gyeom ?

**Comment by cskwak99** [comment]
I have the same problem, too....
If you solve the problem, plz share your answer.

**Comment by Altterisk** [comment]
Friendly reminder that AMP only register sessions with more than 10 packets to the DB. Not sure if this is what you're looking for though.

**Comment by mrpositron** [comment]
Thanks. I already know that. I am sending around 10000 packets.
